src/reynir_correct/errtokenizer.py

```diff
@@ -2604,6 +2607,23 @@ def number_error(
             at_sentence_start = False
 
 
+def unicode_replacement(txt: str) -> str:
```

Comment:
Suggest using re.sub with a function/lambda parameter, cf. spelling.py in GreynirCorrect